6.scDINO_analysis/1.scDINO_run/only_downstream_analyses.yaml

```diff
@@ -25,7 +25,7 @@ compute_cls_features:
   use_cuda: True
   num_gpus: 1
   batch_size_per_gpu: 24
-  num_workers: 10
+  num_workers: 2
```

Comment:
I'm guessing you encountered some reason to reduce the number of workers here. Would it be worth providing a comment as to why?